pkg/api/live/conn.go

```diff
@@ -52,8 +52,12 @@ func (c *connection) readPump() {
 	}()
 
 	c.ws.SetReadLimit(maxMessageSize)
-	c.ws.SetReadDeadline(time.Now().Add(pongWait))
-	c.ws.SetPongHandler(func(string) error { c.ws.SetReadDeadline(time.Now().Add(pongWait)); return nil })
+	if err := c.ws.SetReadDeadline(time.Now().Add(pongWait)); err != nil {
+		log.Warn("Setting read deadline failed: %s", err)
```

Comment:
Need to be `Warnf` to work. But use argument `log.Warn("Failed to write close message to connection", "err", err` instead.

Reply:
@marefr See my above comment.

Reply:
See my above comment